Dockerfile

```diff
@@ -1,21 +1,25 @@
-FROM sonatype/nexus3:3.16.1
+FROM sonatype/nexus3:3.18.1
 MAINTAINER Dwolla Dev <dev+docker-nexus3-crowd@dwolla.com>
 LABEL org.label-schema.vcs-url="https://github.com/Dwolla/docker-nexus3-crowd"
 
 USER root
 COPY startup.sh /
-RUN yum install -y git which && \
-    curl -o /etc/yum.repos.d/epel-apache-maven.repo https://repos.fedorapeople.org/repos/dchen/apache-maven/epel-apache-maven.repo && \
-    yum install -y apache-maven && \
+RUN export M2_HOME=/opt/maven && \
```

Comment:
The yum installation of maven was broken with the upstream switch to use Redhat UBI

Reply:
It looks like it's just called `maven` in RHEL 8, and we get version 3.5 by default without needing EPEL.  To clean up the Dockerfile, could we just add `maven` to the yum install list (and maybe change to using `dnf`?)